.travis.yml

```diff
@@ -223,7 +223,7 @@ jobs:
         - ZMQ_FORCE_SUBPROJECT=true
       
     - stage: trigger dependent repositories
-      if: branch == develop
+      if: branch IN (master, develop)
```

Comment:
So just to make sure I have this clear,  it will trigger, on pushes to master and develop.  And will it trigger on PR's to those branches as well?

Reply:
Yea, triggers on pushes and PRs to the master and develop branches